src/chocolatey/infrastructure.app/services/PowershellService.cs

```diff
@@ -446,6 +446,10 @@ public void PreparePowerShellEnvironment(IPackageSearchMetadata package, Chocola
             Environment.SetEnvironmentVariable("packageTitle", package.Title);
             Environment.SetEnvironmentVariable("chocolateyPackageVersion", package.Identity.Version.ToNormalizedStringChecked());
             Environment.SetEnvironmentVariable("packageVersion", package.Identity.Version.ToNormalizedStringChecked());
+            // We use ToStringSafe on purpose here. There is a need for the version
+            // the package specified, not the normalized version we want users to use.
+            Environment.SetEnvironmentVariable("chocolateyPackageVersionOriginal", package.Identity.Version.ToStringSafe());

```

Comment:
@pauby before moving forward with this PR, what are your thoughts on this variable name? `chocolateyPackageVersionOriginal` This is the potentially non-normalized package version, i.e. 1.2, where as the `chocolateyPackageVersion` variable would contain the normalized version, i.e. 1.2.0.

Reply:
What about `chocolateyPackageVersionPushed`? `Original` doesn't feel right to me and I think will cause confusion.

Reply:
Not sure `Pushed` is right here either. This environment variable doesn't really have anything to do with pushing to package, but rather what was the original version number before it was normalised, I.e. what is the original package version in the nuspec file for the package.

Reply:
> what is the original package version in the nuspec file for the package.

Is that not the version number it was given when it was pushed?

Reply:
When it was `Packed` perhaps, but nothing really to do with when it was `Pushed`.

Reply:
@pauby just thinking about this again... what about `chocolateyPackageVersionFromNuSpec` and `packageVersionFromNuSpec`?

Reply:
I like those. Lets use them.

Reply:
@pauby just had a chat with Kim, and he has made a better suggestion, how about `chocolateyNuSpecVersion` and `nuSpecVersion`?

Reply:
I would prefer `chocolateyPackageNuspecVersion` as it refers to something specific for the package. I can see us publicly documenting this in future, and the `chocolateyPackage` part is consistent with package specific environment variables.

Reply:
After thinking about it and going back and forth on it, I'll get the PR updated to use `chocolateyPackageNuspecVersion`.